decidim-core/app/packs/src/decidim/remote_tooltips.js

```diff
@@ -0,0 +1,31 @@
+import createTooltip from "src/decidim/tooltips"
+
+/**
+ * This function is parsing the DOM and tries to attach and initialize a tooltip starting from an element
```

Comment:
It isn't parsing the DOM in this case, as that's being done in `index.js`, right?

Reply:
Yeah , i meant to say that it gets the element where is attached, tries to find a child node and attach the tooltip to it. Will Change locally

Reply:
Done in [43ddbf1](https://github.com/decidim/decidim/pull/13701/commits/43ddbf1c73378d28bc3eb043987a9389e9aeb914)